js/ui/map.js

```diff
@@ -1059,6 +1090,9 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
      * Destroys the map's underlying resources, including web workers and DOM elements.
      *
      * After calling this method, you must not call any other methods on the map.
+     * @see [Create a timeline animation](https://www.mapbox.com/mapbox-gl-js/example/timeline-animation/)
+     * @see [Display a popup on hover](https://www.mapbox.com/mapbox-gl-js/example/popup-on-hover/)
+     * @see [Filter features within map view](https://www.mapbox.com/mapbox-gl-js/example/filter-features-within-map-view/)
```

Comment:
Got it - removed these. There aren't any corresponding examples for this, so keeping this blank.